Sources/mParticle-Appboy/MPKitAppboy.m

```diff
@@ -878,7 +856,7 @@ - (nonnull MPKitExecStatus *)updateUser:(FilteredMParticleUser *)user request:(F
         }
         
         if (userEmail) {
-            appboyInstance.user.email = userEmail;
+            [appboyInstance.user setEmail:userEmail];
```

Comment:
@einsteinx2  because appboy put deprecation warnings on setting the value that way and recommended the bracket syntax 
<img width="880" alt="Screenshot 2023-02-08 at 9 08 07 AM" src="https://user-images.githubusercontent.com/33703490/217553412-35c30a7e-346f-44c4-a9a1-520cfb879e29.png">
<img width="908" alt="Screenshot 2023-02-08 at 9 08 45 AM" src="https://user-images.githubusercontent.com/33703490/217553471-d2e78f1f-4c65-4451-aba6-517a2a5a5ee6.png">

Reply:
Fair enough :)